scripts/build.d/vim

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRCSYNC=${SRCSYNC:-tarball}
```

Comment:
You are saying to use a tarball by default here.

Reply:
Oh. You're right. I just forgot it.